satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -800,6 +800,7 @@ model graceful_exit_transfer_queue (
     field node_id             blob
     field path                blob
     field piece_num           int
+    field root_piece_id       blob       ( updatable, nullable )
```

Comment:
does it need to be updatable? In what situation would it change?

Reply:
no. i'll remove.